lib/internal/Magento/Framework/App/DeploymentConfig.php

```diff
@@ -170,8 +158,12 @@ private function getEnvOverride() : array
      * @throws FileSystemException
      * @throws RuntimeException
      */
-    private function reloadData(): void
+    private function loadData(): void
     {
+        if (!empty($this->data) && $this->isAvailable()) {
```

Comment:
maybe we can use comparison with null value, it mean it won't trigger not needed load when "data" is empty array?
```suggestion
        if ($this->data! == null && $this->isAvailable()) {
```

Reply:
Applied.